pulpcore/pulpcore/app/models/task.py

```diff
@@ -213,7 +213,7 @@ class Task(Model):
     )
 
     group = models.UUIDField(null=True)
-    state = models.TextField(choices=STATES)
+    state = models.TextField(choices=STATES, null=False)
```

Comment:
According to the [django documentation](https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.Field.null) `null=False` is the default.

Reply:
Ok, that is good.  There's still a weird bug somewhere then, because when I ran a task it gave me the warning on line 247 that the __call__() had occurred but that the ```self.state``` was not "waiting".  I thought maybe it was null (and should have checked before destroying my env).

But I will try to figure out what is going on.